server/index.js

```diff
@@ -0,0 +1,26 @@
+const express = require('express');
+const morgan = require('morgan');
+const db = require('../database/index.js');
+const path = require('path');
+const Listings = require('../database/ListingsNear.js')
+
+const app = express();
+const port = 3010;
+
+app.use(morgan('dev'));
+app.use(express.json())
+// app.use(express.urlencoded({ extended: true }));
+app.use(express.static(path.resolve(__dirname + '/../client/dist')));
+
+app.get('/listings', (req, res) => {
+  Listings.find({})
```

Comment:
Looks like this returns All Listings in your database, rather than a set # of listings. Is that what you want?